testing/nzbget/APKBUILD

```diff
@@ -1,23 +1,22 @@
 # Contributor: August Klein <amatcoder@gmail.com>
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=nzbget
-pkgver=18.1
+pkgver=19.1
 pkgrel=0
 pkgdesc="Usenet downloader"
 url="http://nzbget.net"
 arch="all"
 license="GPL2"
-depends=""
 makedepends="libxml2-dev ncurses-dev libressl-dev"
 subpackages="$pkgname-doc"
-source="https://github.com/nzbget/nzbget/releases/download/v$pkgver/$pkgname-$pkgver-src.tar.gz
-	"
+source="https://github.com/$pkgname/$pkgname/releases/download/v$pkgver/$pkgname-$pkgver-src.tar.gz"
+
 builddir="$srcdir"/$pkgname-$pkgver
 
 prepare() {
 	cd "$builddir"
-	update_config_sub || return 1
-	default_prepare || return 1
+	update_config_sub
+	default_prepare
```

Comment:
Please run `default_prepare` before `cd "$builddir`". I fixed this for you and commited your changes, but please keep this in mind for future patches.